pdc/apps/component/tests.py

```diff
@@ -1866,6 +1866,25 @@ def test_delete_group(self):
         response = self.client.get(url, format='json')
         self.assertEqual(response.status_code, status.HTTP_404_NOT_FOUND)
 
+    def test_list_relationship_by_nested(self):
```

Comment:
Because this feature will affects all apis ordering. So is it ok to add some exceptional test cases like  non-existing-field__name or something